src/resolver.ts

```diff
@@ -65,11 +63,15 @@ export function handleMulticoinAddrChanged(event: AddressChangedEvent): void {
   if(resolver.coinTypes == null) {
     resolver.coinTypes = [coinType];
     resolver.save();
-  } else if(!resolver.coinTypes.includes(coinType)) {
+  } else {
     let coinTypes = resolver.coinTypes
-    coinTypes.push(coinType)
-    resolver.coinTypes = coinTypes
-    resolver.save()
+    if(coinTypes){
```

Comment:
This should always be true, becuase if it were null we would have entered the 'if' instead of the 'else'.